### PR TITLE
New Contrail API - Security Group

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

File: Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1 [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        $this.Name = $Name
+        $this.ProjectName = $ProjectName
+    }
+
+    [String[]] GetFqName() {
+        return @($this.DomainName, $this.ProjectName, $this.Name)
+    }
+
+    static [SecurityGroup] new_Default([String] $ProjectName) {
+        $group = [SecurityGroup]::new('default', $ProjectName)
```
Comment (michal-clapinski) [inline Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1]: Please supply a comment explaining those calls (about allowing all incoming and outgoing packets)

Comment (jacekchalupka) [reply Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1]:
```suggestion
        $group = [SecurityGroup]::new('default', $ProjectName)
        # These two, created below, policy rules
        # allow network traffic to and from security group.
```

File: Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1 [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    SecurityGroup([String] $Name, [String] $ProjectName) {
+        $this.Name = $Name
+        $this.ProjectName = $ProjectName
+    }
+
+    [String[]] GetFqName() {
+        return @($this.DomainName, $this.ProjectName, $this.Name)
+    }
+
+    static [SecurityGroup] new_Default([String] $ProjectName) {
```
Comment (jacekchalupka) [inline Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1]:
```suggestion
    # This method creates a default security group for a project
    # with policy rules that pass all incoming and outgoing traffic.
    static [SecurityGroup] new_Default([String] $ProjectName) {
```

File: Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1 (hunk 1 above)
Comment (jacekchalupka) [reply Test/Utils/ContrailAPI_New/Models/SecurityGroupModel.ps1]: @michal-clapinski Please look at my suggestions about the comments before I commit them (not to trigger useless jobs at CI).

**Pull request merged**